appscale/tools/appscale.py

```diff
@@ -239,10 +239,12 @@ def init(self, environment=None):
     shutil.copy(self.TEMPLATE_APPSCALEFILE, appscalefile_location)
 
 
-  def up(self):
+  def up(self, update=""):
```

Comment:
I would recommend an empty tuple as a default arg here in order to match the expected iterable argument.